src/components/page-active-component/dns-alias.tsx

```diff
@@ -0,0 +1,45 @@
+import * as PropTypes from 'prop-types';
+import { FunctionComponent } from 'react';
+
+import {
+  DNSAliasModel,
+  DNSAliasModelValidationMap,
+} from '../../models/radix-api/applications/dns-alias';
+import { Icon, List, Typography } from '@equinor/eds-core-react';
+import { link } from '@equinor/eds-icons';
+
+export interface DNSAliasProps {
+  dnsAliases?: DNSAliasModel[];
+}
+
+export const DnsAlias: FunctionComponent<DNSAliasProps> = ({ dnsAliases }) => (
+  <>
+    {dnsAliases && (
+      <div className="grid grid--gap-x-small">
+        <Typography variant="h4">
+          DNS alias{dnsAliases.length > 1 ? 'es' : ''}
+        </Typography>
+        <List>
+          {dnsAliases?.map((dnsAlias, index) => (
```

Comment:
unnecessary check, object has been checked prior to usage
```suggestion
          {dnsAliases.map((dnsAlias, index) => (
```